docs/editor/port-forwarding.md

```diff
@@ -0,0 +1,51 @@
+---
+Order: 23
+Area: editor
+TOCTitle: Port Forwarding
+ContentId: d7a80c88-c091-4d13-9240-d432c12407a7
+PageTitle: Port Forwarding Local Services
+DateApproved: 8/3/2023
+MetaDescription: Make your local web services accessible over the internet.
+---
+
+# Local Port Forwarding
+
+Support for port forwarding is built into Visual Studio Code, no extension required. When running a local web service, you can use the **Ports** view to make the service accessible to others over the internet.
+
+## How to use Local Port Forwarding
+
+First, you need to have service you want to forward. If you don't have one yet but do have Node.js installed, you can run this command to start up a server on port 3000:
+
+```
+npx serve
+```
+
+Then, navigate to the **Ports** view, and click **Forward a Port**.
+
+![](./images/port-forwarding/ports-view.png)
+
+If you haven't logged in with Github before, you'll be prompted to log in. Then, enter the port you'd like to forward; the default port using the above command is port 3000. Once you do, the port forwarding system will be started and the **Ports** view will update to show the port you forwarded and its "Forwarded Address."
+
+![](./images/port-forwarding/forwarded-port.png)
+
+Hovering over the Forwarded Address, you can use the inline actions copy the address, open it in your browser, or open an in-editor preview.
+
+By default, the forwarded port is **Private**. When you navigate to the URL, you'll be required to log in with the same Github account you used to start the port forwarding process in VS Code. You can change the visibility right clicking on the port and selecting **Port Visibility > Public**. **Public** ports don't require login.
+
+## Common questions
+
+### How do I forward local services if I'm connected to a remote machine?
+
+Currently, port forwarding only works to expose locally-running services. It doesn't work in remote connections yet, although we expect it to in the future.
+
+### How are forwarded ports secured?
+
+By default, both hosting and connecting to a tunnel requires authentication with the same Github or Microsoft account on each end. In both cases, VS Code will make outbound connections to a service hosted in Azure; no firewall changes are generally necessary, and VS Code doesn't set up any network listeners.
+
+However, if you've made a **Public** port, any user with your link can access the forwarded service. You should be careful to avoid hosting any confidental information or insecure services over such ports.
```

Comment:
nit: "if you've forwarded / opened a Public port" or "if you've made a port public" might flow even better?

```suggestion
However, if you've opened a **Public** port, any user with your link can access the forwarded service. You should be careful to avoid hosting any confidential information or insecure services over such ports.
```